be/src/runtime/primitive_type.h

```diff
@@ -91,6 +91,10 @@ constexpr bool is_string_type(PrimitiveType type) {
     return type == TYPE_CHAR || type == TYPE_VARCHAR || type == TYPE_STRING;
 }
 
+constexpr bool is_variant_string_type(PrimitiveType type) {
```

Comment:
variant string is ambiguous to variant type. can we use is_var_string_type or is_varchar_type